docs/modules/ROOT/pages/module/code.adoc

```diff
@@ -93,10 +93,27 @@ Depicted below is an example of a vulnerability detected in a dependency with a
 
 image::dependency-checker-detailed-report.png[Dependency Checker Report]
 
-To help you resolve these issues recommendations on updates or patches that can address the vulnerabilities are provided along with the relevant advisory links for a more detailed understanding of the issue.
+To help you resolve these issues, recommendations on updates or patches that can address the vulnerabilities are provided along with the relevant advisory links for a more detailed understanding of the issue.
 
 By reviewing and applying the proposed solutions in this report, you can enhance the robustness and reliability of your smart contracts, ensuring adherence to best practices and industry standards. This makes the audit process smoother and improves the preparedness of your contracts for successful deployment.
 
+[[standards]]
+=== Standards
+
+This feature introduces a new section in the report called **Standards**. This section provides insights into your contracts that implement interfaces from the OpenZeppelin Contracts library. For example, if you are using an IERC20, Code Inspector will check the implementation details and properties to ensure you are using this ERC correctly.
+
+#### Detailed Checks
+
+For each function and event defined in an interface, Code Inspector performs comprehensive checks to verify compliance with the standard. These checks include:
+
+- **Signature:** Ensures that the function or event signature matches the standard's specification.
```

Comment:
Here I would change the order to this (since in the list we have this order):
- Signature
- Visibility
- Mutability
- Return type
- Parameter names
- Return names

It would be a good idea to add an example like you did with mutability and visibility, but for the rest of the items.

Another thing is that in events we only check the signature, since they don't return values ​​and don't have any specific visibility/mutability.

So I'll probably add this to the documentation.